src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -293,7 +294,9 @@ export default function ChartExpandedState({ asset }) {
             <SwapActionButton asset={assetWithPrice} color={color} inputType={AssetInputTypes.in} />
           )}
           {hasBalance ? (
-            <SendActionButton asset={assetWithPrice} color={color} fromDiscover={fromDiscover} />
+            isTransferable ? (
+              <SendActionButton asset={assetWithPrice} color={color} fromDiscover={fromDiscover} />
+            ) : null
```

Comment:
I think we can just combine the check `(hasBalance && isTransferable)` here

Reply:
the logic is like this, I didn't want to mess with that
```ts
hasBalance 
  ? isTransferable 
      ? <SendActionButton asset={assetWithPrice} color={color} fromDiscover={fromDiscover} />
      : null
  : swapEnabled 
     ? <SwapActionButton asset={assetWithPrice} ...
     : null
```